ui/src/components/home/ExperienceBiblicalCommunity.js

```diff
@@ -0,0 +1,468 @@
+import {
+  Box,
+  Container,
+  Flex,
+  HStack,
+  Text,
+  VStack,
+  Heading,
+  Button,
+  Fade,
+  Spacer,
+  Image,
+  useMediaQuery,
+  Link,
+} from '@chakra-ui/react';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import { ArrowForwardIcon } from '@chakra-ui/icons';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import NewHereForm from './NewHereForm';
+import { useState } from 'react';
+import { InView } from 'react-intersection-observer';
+
+const ExperienceBiblicalCommunity = () => {
+  const [isMobile] = useMediaQuery('(max-width: 768px)');
+  const [isOpen, setIsOpen] = useState(false);
+  const onClose = () => setIsOpen(false);
+  const onOpen = () => setIsOpen(true);
+  const settings = {
+    infinite: true,
+    autoplay: true,
+    autoplaySpeed: 3000,
```

Comment:
in the original design demo this carousel was continuously moving? this demo is quite slow the movement. not sure if this is what the designers wanted, maybe can check with them. if they want it to continuously move, i think you just need to make the `speed` larger than the `autoplaySpeed`

also `css-ease:linear` will make the speedup and slowdown more smooth

Reply:
changed the speed, think maybe best to release to uat first to see which one design team prefers